stage1/rootfs/aggregate/units/prepare-app@.service

```diff
@@ -2,6 +2,8 @@
 Description=Prepare minimum environment for chrooted applications
 DefaultDependencies=false
 OnFailureJobMode=fail
+Requires=systemd-journald.service
```

Comment:
Yep. It's present when you use RKT_STAGE1_USR_FROM=src but if we switch to the coreos image we should add it.